my-sms/app/helpers/institutes_helper.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+module InstitutesHelper
+end
```

Comment:
Used for what exactly?
If you have a plan to use the file, can leave a TODO
`TODO: helper methods coming in next Institute feature for ... blah`

Otherwise its a waste of space and can be removed

Reply:
Removed in e5bb69888691df727165c5761d9a85cd7c3e351c